assets/quick-order-list.js

```diff
@@ -405,6 +407,11 @@ class QuickOrderList extends HTMLElement {
     this.updateLiveRegions(id, message);
   }
 
+  cleanErrors() {
+    this.querySelectorAll('.desktop-row-error').forEach((error) => error.classList.add('hidden'));
+    this.querySelectorAll(`.variant-item__error-text`).forEach((error) => error.innerHTML = '');
```

Comment:
There are a few reasons:

1. I believe that a proper way is not just to hide an element where am error occurs but also clean a message.
2. For desktop we can hide a row where the message is living because there is nothing else there but for mobile the error message is living together with other elements so we should hide particularly the message element or just clean it as I do. Anyways, there would be two different hiding approach.